### DISCUSSION TITLE
hasura seed create ... --from-table, syntax of 'stringArray' parameter ?

### DISCUSSION THREAD

**Discussion by jfalch** [hasura/graphql-engine#8543] [Question] [answered]
hasura cli help says that param to _seed create_ option _--from-table_ may be a 'stringArray'. What format to use for this ? Tried "table1, table2", "[table1, table2]", and more, none of which works. How to specify if I want to use more than one table with the --from-table option ? (Yes, I know about schema. prefix and --f-t t1 --f-t t2 alternative.)

**Comment by adas98012** [comment, marked as answer]
Please continue to use the  "--f-t t1 --f-t t2 alternative", our Documentation team will update the document with regards to the stringArray option. Thank you for bringing to our attention.
